src/reflect/scala/reflect/internal/SymbolTable.scala

```diff
@@ -479,8 +479,22 @@ abstract class SymbolTable extends macros.Universe
     def transform(sym: Symbol, tpe: Type): Type = tpe
   }
   /** The set of all installed infotransformers. */
-  def infoTransformers = Parallel.synchronizeAccess(this){_infoTransformers}
-  def infoTransformers_=(v: InfoTransformer) = Parallel.synchronizeAccess(this){_infoTransformers = v}
+  def infoTransformers = Parallel.synchronizeAccess(this) {
+    val start = if(StatisticsStatics.areSomeColdStatsEnabled) statistics.pushTimer(statistics.symbolOpsStack, statistics.infoTransformersNanos) else null
```

Comment:
@romanowski was mentioning to me that some of that blocks are re-entrant. You need some flag to make sure you are not measuring time in sub-sequent blocks multiple times.

Reply:
Or is it solved by stackable timer?